cmd/ftl/cmd_serve.go

```diff
@@ -91,6 +106,77 @@ func (s *serveCmd) Run(ctx context.Context) error {
 	return nil
 }
 
+func runInBackground(ctx context.Context, logger *log.Logger) {
+	pidFilePath := pidPath(logger)
+	if _, err := os.Stat(pidFilePath); err == nil {
+		logger.Warnf("'ftl serve' is already running in the background. Use --stop to stop it.")
+		return
+	}
+
+	args := make([]string, 0, len(os.Args))
+	for _, arg := range os.Args[1:] {
+		if arg == "--background" {
+			continue
+		}
+		args = append(args, arg)
+	}
+
+	cmd := exec.Command(ctx, logger.GetLevel(), "", os.Args[0], args...)
```

Comment:
I think it would be better to use `os/exec` rather than our wrapper, as we don't need or want anything that it adds.